plugins/woocommerce/includes/class-wc-install.php

```diff
@@ -247,6 +247,7 @@ class WC_Install {
 		),
 		'8.7.0' => array(
 			'wc_update_870_prevent_listing_of_transient_files_directory',
+			'wc_update_870_add_launch_your_store_tour_option',
```

Comment:
I think this needs to be in `8.9.0` instead

Reply:
Updated in [woocommerce_launch_your_store_tour_hidden should be added < 8.9](https://github.com/woocommerce/woocommerce/pull/46203/commits/7bbf8c1c03f1784f3ead2e0a3a333b99ffd647ce)